disperser/batcher/txn_manager.go

```diff
@@ -192,8 +198,31 @@ func (t *txnManager) ReceiptChan() chan *ReceiptOrErr {
 	return t.receiptChan
 }
 
+// ensureAnyTransactionBroadcasted waits until all given transactions are broadcasted to the network.
+func (t *txnManager) ensureAnyTransactionBroadcasted(ctx context.Context, txs []*transaction) error {
+	queryTicker := time.NewTicker(queryTickerDuration)
+	defer queryTicker.Stop()
+
+	for {
+		for _, tx := range txs {
+			_, err := t.wallet.GetTransactionReceipt(ctx, tx.TxID)
+			if err == nil || errors.Is(err, walletsdk.ErrReceiptNotYetAvailable) {
```

Comment:
Checking error types seems like it's a bit brittle. I guess there's no other option here?

Reply:
We can change the implementation in eigensdk, but I think this signature should be kept. 
One thing we can do is make the error type more expressive and return Fireblocks status as part of the error, but not sure if that will be less brittle

Reply:
Maybe add a comment: "Before the transaction is broadcast, this wallet method will return error of type `walletsdk.Whatever`. Once the transaction has been broadcast, it will either successfully return the receipt or return an error of type `walletsdk.ErrReceiptNotYetAvailable`."